tests/infra/base_smoketest.py

```diff
@@ -183,7 +183,12 @@ def _test_get_subscriptions(self, replica, requested_subscription):
         list_of_subscription_uuids = [x['uuid'] for x in list_of_subscriptions if x['uuid']]
         self.assertIn(requested_subscription, list_of_subscription_uuids)
 
-    def _test_replica_sync(self, current_replica, bundle_uuid, ):
+    def _download_bundle(self, replica_name: str, bundle_uuid: str):
+        with tempfile.TemporaryDirectory(prefix=f'{self.workdir.name}/') as tempdir:  # type: ignore
+            run(f"{self.venv_bin}hca dss download --replica {replica_name} --bundle-uuid {bundle_uuid}"  # type: ignore
```

Comment:
Can we get rid of the `type: ignore`s?

Reply:
MyPy says that `self.workdir` and `self.venv_bin` don't exist.